box2d/native/ext.manifest

```diff
@@ -1 +1,95 @@
-name: "box2d"
\ No newline at end of file
+name: "box2d"
```

Comment:
We can specify everything we need to exclude or include right here. So you don't need to manually install app manifest anymore